content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -174,6 +212,14 @@ Use the following commands to install Containerd on your system:
 
 On Linux the default CRI socket for containerd is `/run/containerd/containerd.sock`.
 On Windows the default CRI endpoint is `npipe://./pipe/containerd-containerd`.
+If you are running Windows, you might want to exclude containerd from Windows Defender Scans
```

Comment:
I did not add these lines, i need to find out how this change made it in here

Reply:
probably due to the rebase. you can just remove these changes from 214 to 221.

Reply:
Done